bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -200,4 +209,233 @@ private void createComplete(int rc, LedgerHandle lh) {
         cb.createComplete(rc, lh, ctx);
     }
 
+    private static final byte[] EMPTY_PASSWORD = new byte[0];
+
+    public static class CreateBuilderImpl implements CreateBuilder {
```

Comment:
why it is public? it should be package private.

Reply:
It was because initially I had the tests cases in another package, now that tests are in o.a.b.client there is no need.
fixed